test/golden/library_example.golden

```diff
@@ -44,26 +44,26 @@
 /java/google-cloud-library_example/.gradle/2.12/taskArtifacts/outputFileStates.bin
 /java/google-cloud-library_example/.gradle/2.12/taskArtifacts/taskArtifacts.bin
 /java/google-cloud-library_example/build.gradle
-/java/google-cloud-library_example/build/classes/main/com/google/cloud/example/library/spi/v1/LibraryServiceClient.class
```

Comment:
Did you intend to update the golden file while you disable the test?

Reply:
Yes. Because the same golden test is running in another circleci task.